proto/intertx/tx.proto

```diff
@@ -22,6 +22,7 @@ message MsgRegisterAccount {
 
   string owner                      = 1;
   string connection_id              = 2 [(gogoproto.moretags) = "yaml:\"connection_id\""];
+  string version                    = 3 [(gogoproto.moretags) = "yaml:\"version\""];
```

Comment:
I think you can omit the yaml tags for this field! :)

Reply:
Done